README.md

```diff
@@ -129,7 +129,7 @@ Example command:
 
```

Comment:
![image](https://github.com/evillaz/school-library/assets/93039359/01aa2f64-de26-474b-a75e-e75ca5d3abeb)

- [ ] It would be nice if you fix the highlighted links in the image above to make your readme descriptive and professional 👍 because right now these links are broken 😨